templates/zerver/integrations/github-actions.md

```diff
@@ -1,8 +1,25 @@
-It's easy to send Zulip messages from GitHub Actions! This is useful:
+# Zulip GitHub Actions integration
 
-  * If you want to report the success or failure of an Action to Zulip.
-  * If you want to send output from your actions to Zulip.
+It's easy to send Zulip messages from GitHub Actions!
 
-Instructions are available [here](https://github.com/zulip/github-actions-zulip#readme).
+{start_tabs}
 
-See also the [GitHub integration](/integrations/doc/github).
+1. {!create-a-generic-bot.md!}
+
+1. To send Zulip notifications whenever your workflow runs, add the
+   `zulip/github-actions-zulip/send-message@v1` action to your GitHub
+   Actions workflow file, and set the input values as specified in the
+   [README][README] of the [Zulip GitHub Actions repository][repo].
+
+{end_tabs}
+
+### Related documentation
+
+* [Zulip GitHub Actions repository][repo]
```

Comment:
Since the link to the repo was added into the second instruction (in the last update), this link has been edited to re-use it, by adding the 'repo' ref link.